splitpdb/src/splitpdb_new/src/splitpdb_new/IO.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Let us discuss whether we still need the code duplicates splitpdb and splitpdb_new.

Reply:
I deleted the duplicates and have pushed the new commit.